runelite-api/src/main/java/net/runelite/api/widgets/WidgetID.java

```diff
@@ -167,6 +167,7 @@
 	public static final int TEMPOROSS_GROUP_ID = 437;
 	public static final int CLAN_GROUP_ID = 701;
 	public static final int CLAN_GUEST_GROUP_ID = 702;
+	public static final int TOB_HEALTH_BAR_GROUP_ID = 28;
```

Comment:
There's already a `TOB_PARTY_GROUP_ID` defined as `28`. Ideally we shouldn't define the group ID twice, so maybe rename that `TOB_GROUP_ID`.

Reply:
Agreed. That widget actually has more than just party info. It has health info (as mentioned above in this addition), and also it's the same widget that displays the big red text on-screen that scrolls up (like the boss name, or "Seeking Party..." etc.)

Reply:
Can tell how much my brain was frazzled yesterday :d